docs/docs/code/nodejs/sharing-code/README.md

```diff
@@ -5,57 +5,89 @@ thumbnail: https://res.cloudinary.com/pipedreamin/image/upload/v1646841235/docs/
 
 # Sharing code across workflows
 
+[Actions](/components#actions) are reusable steps. When you author an action, you can add it to your workflow like you would other actions, by clicking the **+** button below any step.
+
 Pipedream provides two ways to share code across workflows:
 
-- **Create an action**. [Actions](/components#actions) are reusable steps. When you author an action, you can add it to your workflow like you would other actions, by clicking the **+** button below any step. [Learn how to build your first action here](/components/quickstart/nodejs/actions/).
-- **Create your own npm package**. If you need to run the same Node.js code in multiple workflows, you can publish that code as an npm package. We'll walk you through that process below.
+- **Publish an action from a Node.js code step**. [Publish any Node.js code step as a reusable action](/code/nodejs/sharing-code/#publish-an-action-from-a-node-js-code-step) from the Pipedream dashboard.
+
+- **Create an action from code**. Develop your action code on your local filesystem and [publish to your Pipedream account using the Pipedream CLI](/components/quickstart/nodejs/actions/).
 
-[[toc]]
+## Publish an action from a Node.js code step
 
-## Publishing your own npm package
+You can publish any of Node.js code steps into a reusable action. This enables you to write a Node.js code step once, and reuse it across many workflows without rewriting it.
```

Comment:
`You can publish any of **your** Node.js code step` or `You can publish any Node.js code step`?